ios/Podfile.lock

```diff
@@ -13,10 +13,10 @@ PODS:
   - glog (0.3.5)
   - React (0.59.0):
     - React/Core (= 0.59.0)
-  - react-native-bpk-component-calendar (0.3.14):
+  - react-native-bpk-component-calendar (1.0.0):
     - Backpack (~> 10.0)
     - React
-  - react-native-bpk-component-dialog (0.1.0):
+  - react-native-bpk-component-dialog (1.0.0):
```

Comment:
This along with lockfiles have changed because they should have been done on `master` already.